### DISCUSSION TITLE
Docker image url doesn't work for me

### DISCUSSION THREAD

**Discussion by Partysun** [toeverything/AFFiNE#688] [Q&A] [answered]
Hi ✋
I can't pull this image by 
`docker pull ghcr.io/toeverything/affine:nightly-server-latest`
Does this link work or it's smith on my side?

Thank you

**Comment by Partysun** [comment, marked as answer]
That link works
```
docker pull ghcr.io/toeverything/affine-self-hosted:latest
```
Thank you

**Reply by doodlewind** [reply]
Thanks for your sharing!

**Comment by Yipei-Operation** [comment]
The docker image of the downhills version is still under developing, for now, can refer to this guide:https://community.affine.pro/c/user-guide/self-host-affine-with-docker-pre-alpha-and-alpha-local-setup-user-guide
